lasagne/layers/shape.py

```diff
@@ -98,8 +95,8 @@ def get_output_shape_for(self, input_shape, *args, **kwargs):
                                      (o[0], len(input_shape)))
                 output_shape[dim] = input_shape[o[0]]
                 masked_output_shape[dim] = input_shape[o[0]]
-                if ((input_shape[o[0]] is None)
-                    and (masked_input_shape[o[0]] is None)):
+                if (input_shape[o[0]] is None) \
+                   and (masked_input_shape[o[0]] is None):
```

Comment:
Yes, absolutely I'm using parentheses preferentially over line continuation, but see my example above -- when there's an `if` statement with parentheses, it causes a different PEP-8 issue, which is the only place I use line continuation.